operators/advanced_network/advanced_network/common.cpp

```diff
@@ -15,8 +15,8 @@
  * limitations under the License.
  */
 
-#include "adv_network_mgr.h"
-#include "adv_network_common.h"
+#include "./manager.h"
+#include "./common.h"
```

Comment:
`./` seems unusual, is this required?

Reply:
not required, just makes it explicit we're dealing with headers in the current project/folder so the generic name by itself doesn't feel like it could be anything from anywhere.

Reply:
I would prefer to either have longer include names with the paths or make the file names longer

Reply:
Sounds good. The new change should meet that.